force-app/main/default/pages/GE_TokenizeCard.page

```diff
@@ -101,11 +102,11 @@
 
                 sfdo.mount(config)
                     .then(() => {
-                        const successMessage = JSON.stringify({ type: postMessageType, success: true });
+                        const successMessage = { type: postMessageType, success: true };
```

Comment:
@lparrott these are the changes that. were made for the credit card widget to load properly again, I've been debating writing a function that checks if an object is stringified and then parses it conditionally but I'm not sure if that is really worth it in this case, your feedback would be much appreciated here!

Reply:
We hopefully won't need that, as we should be consistent in how this gets called, but we may find a place that we missed during testing!